packages/common/src/jsonApiClient/index.js

```diff
@@ -0,0 +1,38 @@
+const { createApiClient } = require('../apiClient')
+
+module.exports = function createJsonApiClient({
+  apiConfigInput,
+  createEndpoint,
+}) {
+  const apiClient = createApiClient(apiConfigInput)
+  const enpointConfigCache = {}
+  const call = (input, userInput) => {
+    let enpointConfig
```

Comment:
=> endpointConfig (think it's worth changing so it shows up when searching in the repo for `endpointConfig`

Reply:
haha true